### PR TITLE
193AddedPD_VERIFIED column

### PR DESCRIPTION
**Pull request by 2011sandeepraj**
Added pd_verified to the database

### CODE REVIEW

File: docs/howto/db-migration.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+   $ npx typeorm migration:create -n "addStudentPDColumn"
+   ```
+3. The above command will generate a migration file containing method stubs for async up and async down methods under the following package
+   `sources/packages/api/src/database/migrations`
+
+4. Add the sql that is intended to be run using
+   `queryRunner.query()` method inside up method.
+   See e.g. in `SIMS\sources\packages\api\src\database\migrations\1614107103138-StudentPDVerified.ts\1614107103138-StudentPDVerified.ts`
+
+5. If applicable Add the sql that is intended to be run using
```
Comment (andrewsignori-aot) [inline docs/howto/db-migration.md:16]: I would recommend removing the "If applicable". We should always add a rollback. Not adding would be a major exception scenario.

File: docs/howto/db-migration.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+   queryRunner.query() method inside down method. This will be run when database is being wiped out.
```
Comment (andrewsignori-aot) [inline docs/howto/db-migration.md:17]: The rollback runs when the migration is "rolled back", not necessarily during a database cleanup.

File: docs/howto/db-migration.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+   See e.g. in `SIMS\sources\packages\api\src\database\migrations\1614107103138-StudentPDVerified.ts\1614107103138-StudentPDVerified.ts`
+
+6. Run the command to clean out all the volumes(database)
+
+   ```bash
+   $ make local-clean
```
Comment (andrewsignori-aot) [inline docs/howto/db-migration.md:23]: That is only necessary if we want to test all the migrations at once. To test the particular one, we can just run the application with the "make-local".

File: sources/packages/api/src-sql/StudentPDVerified/Add-StudentPDVerified.sql [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
+-- Add the column with pd_verified.
+ALTER TABLE students
+    ADD COLUMN IF NOT EXISTS pd_verified BOOLEAN DEFAULT ;
+   
+-- -- Removing the dummy constraint created just to allow us
```
Comment (andrewsignori-aot) [inline sources/packages/api/src-sql/StudentPDVerified/Add-StudentPDVerified.sql:6]: Please remove the comment code.

File: sources/packages/api/src-sql/StudentPDVerified/Add-StudentPDVerified.sql [hunk 5]
```diff
@@ -0,0 +1,13 @@
+
+-- Add the column with pd_verified.
+ALTER TABLE students
```
Comment (andrewsignori-aot) [inline sources/packages/api/src-sql/StudentPDVerified/Add-StudentPDVerified.sql:3]: I believe that there is a problem with the syntax.
The correct command would be:
ALTER TABLE students
    ADD COLUMN IF NOT EXISTS pd_verified BOOLEAN;

**Review by andrewsignori-aot — CHANGES_REQUESTED** [review]
Really nice working creating the documentation 👍 
Please take a look at the comments and ensure that the migration script is working as expected.

**Review by andrewsignori-aot — APPROVED** [review]
After a chat with @2011sandeepraj he updated the "up" script with the correct file and it looks good, hence approving this PR.

**Review by fwpushan — APPROVED** [review]

**Pull request merged**